SimGeneral/Debugging/python/CfiFile_cfi.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+demo = cms.EDAnalyzer('Debugging'
```

Comment:
this file seems unnecessary... a CFI for the debugger will be generated automatically by the filldescriptions